Snowflake.Data.Tests/UnitTests/QueryContextCacheTest.cs

```diff
@@ -0,0 +1,270 @@
+﻿/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    using Newtonsoft.Json;
+    using NUnit.Framework;
+    using Snowflake.Data.Core;
+    using System;
+    using System.Collections.Generic;
+
+    [TestFixture]
+    class QueryContextCacheTest
+    {
+        private QueryContextCache qcc = null;
+        private long BASE_READ_TIMESTAMP = 1668727958;
+        private string CONTEXT = "Some query context";
+        private long BASE_ID = 0;
+        private long BASE_PRIORITY = 0;
+
+        private int MAX_CAPACITY = 5;
```

Comment:
const variables should have PascalCasing

Reply:
done